### PR TITLE
[DOCUMENTATION] Add `psql` prereq. to `building.md`

### PR DESCRIPTION
**Pull request by pcrosemurgy**
* **Tickets addressed:** None
* **Review:** By commit  
* **Merge strategy:** Merge (no squash)  

## Description
Adds `psql` prereq. to `building.md`. This is required to be installed locally since #11 was merged. 

## Documentation
Updated `building.md`.

### CODE REVIEW

File: docs/building.md [hunk 1]
```diff
@@ -6,10 +6,11 @@
 
 ## Prerequisites
 
-| Name    | Version          | Notes        | Download              |
-| ------- | ---------------- | ------------ | --------------------- |
-| OpenJDK | Temurin 17 (LTS) | HotSpot JVM  | https://adoptium.net/ |
-| Gradle  | 7.2              | Build system | https://gradle.org    |
+| Name       | Version          | Notes                      | Download              |
+| ---------- | ---------------- | ------------               | --------------------- |
```
Comment (Twisol) [inline docs/building.md]: Minor, but formatting:
```suggestion
| ---------- | ---------------- | -------------------------- | --------------------- |
```

**Review by Twisol — APPROVED** [review]
Idle thought, but Gradle is not required for runtime, while PostgreSQL is not required at build time -- would it make any sense to separate (or at least mark) build vs. runtime requirements?

File: docs/building.md (hunk 1 above)
Comment (pcrosemurgy) [reply docs/building.md]: Pushed! Thanks

**Comment by pcrosemurgy** [comment]
@Twisol For sure, latest push separates build vs. testing prereqs.

**Review by goetzrrGit — APPROVED** [review]
Looks good

**Review by kroffo — APPROVED** [review]
Just this morning I found this requirement out the hard way. I've had psql installed, just not added to my PATH! Thanks for taking the initiative on this Paul!

**Review by mattdailis — APPROVED** [review]

**Pull request merged**